scripts/helpersDocs.md

```diff
@@ -0,0 +1,1050 @@
+        <td>Always</td>
+        <td>
+            Used when you want to have a custom query params for the http call.
+        </td>
+    </tr>
+    <tr>
+        <td>Body</td>
+        <td>json</td>
+        <td>no</td>
+        <td> - </td>
+        <td>Always</td>
+        <td>
+            A payload of data can be sent to the server in the body of the request.
+        </td>
+    </tr>
+    <tr>
+        <td>Override Settings</td>
+        <td>boolean</td>
+        <td>no</td>
+        <td> false </td>
+        <td>Always</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Follow Redirect</td>
+        <td>boolean</td>
+        <td>no</td>
+        <td> false </td>
+        <td> overrideSettings </td>
+        <td>Indicates that the resource has to be downloaded into a file instead of returning it in the response.</td>
+    </tr>
+    <tr>
+        <td>Download</td>
+        <td>boolean</td>
+        <td>no</td>
+        <td> false </td>
+        <td> overrideSettings </td>
+        <td>If true the method won't return until the file has been downloaded, and it will return all the information of the file.</td>
+    </tr>
+    <tr>
+        <td>File name</td>
+        <td>text</td>
+        <td>no</td>
+        <td></td>
+        <td> overrideSettings </td>
+        <td>If provided, the file will be stored with this name. If empty the file name will be calculated from the URL.</td>
+    </tr>
+    <tr>
+        <td>Full response</td>
+        <td> boolean </td>
+        <td>no</td>
+        <td> false </td>
+        <td> overrideSettings </td>
+        <td>Include extended information about response</td>
+    </tr>
+    <tr>
+        <td>Connection Timeout</td>
+        <td> number </td>
+        <td>no</td>
+        <td> 5000 </td>
+        <td> overrideSettings </td>
+        <td>Connect timeout interval, in milliseconds (0 = infinity).</td>
+    </tr>
+    <tr>
+        <td>Read Timeout</td>
+        <td> number </td>
+        <td>no</td>
+        <td> 60000 </td>
+        <td> overrideSettings </td>
+        <td>Read timeout interval, in milliseconds (0 = infinity).</td>
+    </tr>
+    </tbody>
+</table>
+
+<h3>Outputs</h3>
+
+<table>
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>response</td>
+        <td>object</td>
+        <td>
+            Object resulting from the response to the endpoint call.
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+
+</details>
+
+For more information about how shortcuts or flow steps works, and how they are generated, take a look at the [slingr-helpgen tool](https://github.com/slingr-stack/slingr-helpgen).
+
+## Additional Flow Step
+
+
+<details>
+    <summary>Click here to see the Customs Flow Steps</summary>
+
+<br>
+
+
+
+### Custom Flow Steps Name
+
+Description of Custom Flow Steps
+
+*MANUALLY ADD THE DOCUMENTATION OF THESE FLOW STEPS HERE...*
+
+
+</details>
```

Comment:
Add custom flow step documentation

Reply:
The documentation of the customs steps is in the Readme file, it just remains as a generated file.